livekit-agents/livekit/agents/utils/codecs/decoder.py

```diff
@@ -136,20 +143,27 @@ def _decode_loop(self):
                         rtc.AudioFrame(
                             data=data,
                             num_channels=nchannels,
-                            sample_rate=resampled_frame.sample_rate,
-                            samples_per_channel=resampled_frame.samples / nchannels,
+                            sample_rate=int(resampled_frame.sample_rate),
```

Comment:
nit: btw we could use the rtc.AudioResampler instead

Reply:
this is basically the same thing.. exposed by libav